SgmlTests/Tests-Logic.cs

```diff
@@ -9,143 +9,168 @@
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
 
-using System;
-using System.IO;
-using System.Xml;
 using log4net;
 using NUnit.Framework;
 using Sgml;
+using System;
+using System.IO;
+using System.Xml;
 
-namespace SGMLTests {
-    public partial class Tests {
-
-        //--- Types ---
+namespace SGMLTests
+{
+    public partial class Tests
+    {
         private delegate void XmlReaderTestCallback(XmlReader reader, XmlWriter xmlWriter);
 
-        private enum XmlRender {
+        private enum XmlRender
+        {
             Doc,
             DocClone,
             Passthrough
         }
 
-        //--- Class Fields ---
-        private static ILog _log = LogManager.GetLogger(typeof(Tests));
-        private static bool _debug = true;
```

Comment:
I replaced this `static readonly bool _debug` with `#if DEBUG` below btw.

Reply:
Yep saw that.  I'm not sure I even what all that debug output with a debug build of sgml reader...  it was more an internal flag that I use only for very specific type of debugging SgmlReader.  Perhaps it is ok to make that #if debug though since I rarely use debug SgmlReader for anything else.  But if I'm debugging SgmlReader with various debug outputs, this new debug output might become a problem that obscures the debug output I really want...  but we can cross that bridge if it happens down the road...